python/taichi/__init__.py

```diff
@@ -7,7 +7,7 @@
 from taichi.misc import *
 from taichi.misc.gui import GUI
 from taichi.misc.np2ply import PLYWriter
-from taichi.misc.image import imread, imwrite, imshow
+from taichi.misc.image import *
```

Comment:
nit: Since we are not using either `__all__` or [`name wrangling`](http://google.github.io/styleguide/pyguide.html#3162-naming-conventions) for functions, it would be great to follow `import this` and [PEP](https://legacy.python.org/dev/peps/pep-0328/#id7) to try to avoid `import *` wherever possible :P